modules/sdk-lib-mpc/src/tss/ecdsa-dkls/dkg.ts

```diff
@@ -77,9 +131,10 @@ export class Dkg {
   }
 
   getKeyShare(): Buffer {
-    const keyShareBuff = Buffer.from(this.dkgKeyShare.toBytes());
-    this.dkgKeyShare.free();
-    return keyShareBuff;
+    if (!this.keyShareBuff) {
+      throw Error('Can not get key share, DKG is not complete yet.');
+    }
+    return this.keyShareBuff;
```

Comment:
Is removing `this.dkgKeyShare.free();` intentional?

Reply:
moved it somewhere else so not a removal